webpack/assets/javascripts/react_app/components/common/CardTemplate/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import CardItem from './CardItem';
+
+import { addGlobalFill } from '../../common/Fill/GlobalFill';
+
+const CardTemplate = ({ content, header, slotID, cardID }) => {
```

Comment:
We are not using the CardTemplate anywhere. Is it just for reference? If so, I think we should move it to some different folder and use it in a StoryBook?

Reply:
we use this`cardItem` component in plugins, i.e - https://github.com/theforeman/foreman_remote_execution/pull/549

Reply:
Nope, I see the `CardItem` used there :thinking:

Reply:
You're right, it's probably a leftover :)
deleted